src/AuthorizedSocketConnection.js

```diff
@@ -8,21 +8,9 @@ import {
 } from 'graphql';
 import IoServer from 'socket.io';
 
-import * as asyncUtils from './asyncUtils';
+import * as AsyncUtils from './AsyncUtils';
```

Comment:
didn't @itajaja want this the other way around

Reply:
idk, i went with the convention i use everywhere :P it's whatever

Reply:
upper case should only be for classes imo, or at least that's what we do, no? I don't care much anyway

Reply:
the React-y convention is PascalCase for namespaces,/modules that only have named exports. That's what we do in QSI. I also don't care that much

Reply:
:+1: i think this is the right convention

Reply:
i mean "what @jquense says"